data/migrations/20181015142904_users.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function(knex) {
+    return knex.schema.createTable('users', users => {
+      users.increments();
```

Comment:
Personal preference here, but I like to put whitespace between each field for better readability.